### PR TITLE
Daniel/eng 1780 platform demo updates for relaxed embedded session

### PR DESCRIPTION
**Pull request by danielwong2268**

### CODE REVIEW

File: components/EditableDateField/index.tsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
-import InputMask, { Props } from 'react-input-mask'
+import { PatternFormat } from 'react-number-format'
```
Comment (danielwong2268) [inline components/EditableDateField/index.tsx:1]: react-input-mask was not working with react strict mode. this lib is no longer supported and can be replaced by react-number-format. we just made this change to KanmonOS.

File: pages/api/create_embedded_sessions.ts [hunk 2]
```diff
@@ -114,6 +115,59 @@ const getSessionInvoiceWithFileConnectSessionTokenData = async (
...
+    data: {
+      component: 'SESSION_INVOICE_FLOW_RELAXED',
+      invoices: invoices.map((invoice) => {
+        const sessionInvoice: SessionInvoice = {
+          payorType: invoice.payorType,
+          platformInvoiceId: invoice.id,
+          platformInvoiceNumber: invoice.invoiceNumber,
+          invoiceAmountCents: getInvoiceTotalCents(invoice, true),
+          invoiceIssuedDate: invoice.createdAtIsoDate,
+          invoiceDueDate: invoice.dueDateIsoDate as string,
```
Comment (yaokanmon) [inline pages/api/create_embedded_sessions.ts:133]: Do you need the casting here?

**Review by yaokanmon — APPROVED** [review]
One nit, LGTM

File: pages/api/create_embedded_sessions.ts (hunk 2 above)
Comment (danielwong2268) [reply pages/api/create_embedded_sessions.ts]: i can remove it. the whole object is being casted to any anyway

**Pull request merged**